src/Config/upiconfig.xml

```diff
@@ -14,7 +14,7 @@
             <upi_attribute name='id' value='Win64'/>
             <upi_element name='level'>
               <upi_attribute name='name' value='build'/>
-              <upi_attribute name='id' value='3.6.0' />
+              <upi_attribute name='id' value='3.6.1' />
```

Comment:
@QilongTang confirming that, this change is only needed under build attribute and for release id(in line 11).
Because for some previous patch release branches we did in both locations but looking at 3.4.2 and 3.5.2, only this was changed by you. Will update wiki once you confirm.

Reply:
It should be never needed in the first place, UPI distinguish a release v.s. an update